packages/hoppscotch-common/src/components/environments/teams/index.vue

```diff
@@ -45,8 +52,20 @@
     </div>
     <HoppSmartPlaceholder
       v-if="
+        filterTexts &&
+        searchedAndAlphabeticallySortedTeamEnvironments.length === 0
+      "
+      :alt="`${t('empty.search_environment')}`"
+      :text="`${t('empty.search_environment')} '${filterTexts}'`"
+    >
+      <template #icon>
+        <icon-lucide-search class="svg-icons opacity-75" />
+      </template>
+    </HoppSmartPlaceholder>
+    <HoppSmartPlaceholder
+      v-else-if="
         !loading &&
-        !alphabeticallySortedTeamEnvironments.length &&
+        !searchedAndAlphabeticallySortedTeamEnvironments.length &&
```

Comment:
The markup can be unified similar to the case above. The loading state element can probably be moved to the top, avoiding the need to include negated checks `!loading` in the conditional blocks.